src/network/jwt.ts

```diff
@@ -0,0 +1,69 @@
+import { useDispatch, useSelector } from "react-redux";
+import { selectJwt, actions } from "../stores/jwts";
+
+export interface Jwt {
+  readonly encoded: string;
+  readonly username: string;
+  readonly iat: number;
+  readonly exp: number;
+}
+
+export function parseJwt(token: string): Jwt {
+  var base64Url = token.split(".")[1];
+  var base64 = base64Url.replace(/-/g, "+").replace(/_/g, "/");
+  var jsonPayload = decodeURIComponent(
```

Comment:
I forgot to replace `var` with `const`. This code is taken from [here](https://stackoverflow.com/a/38552302/3393058).